doc/api/net.md

```diff
@@ -331,10 +331,18 @@ Returns `server`.
 added: v0.3.4
 -->
 
-This object is an abstraction of a TCP or local socket.  `net.Socket`
-instances implement a duplex Stream interface.  They can be created by the
-user and used as a client (with [`connect()`][]) or they can be created by Node.js
-and passed to the user through the `'connection'` event of a server.
+This class is an abstraction of a TCP socket, a UNIX domain socket, or
+a Windows named pipe. A `net.Socket` is also a [duplex stream][], so it
```

Comment:
consider: `a abstraction of a TCP or local streaming socket (local uses named pipes on Windows, and UNIX domain sockets otherwise)` ... this long list of underlying technologies seemed random to you at first, I think it will confuse others, and it would be good to emphasize the OS mechanism being abstracted is "local streaming IPC", it just happens to have different names on UNIX and Windows (the Windows O/S devs weren't about to call a core IPC mechanism "unix" I guess ;-).